src/User/UpdateProfile.jsx

```diff
@@ -1,17 +1,82 @@
 import React from "react";
-import {
-  StyledForm,
-  StyledCard
-} from "../StyledFormComponents";
+import Input from "../Input";
+import { useHistory } from "react-router-dom";
+import { StyledForm, StyledCard, StyledCardHeading } from "../StyledFormComponents";
+import { updateUserById } from "../api";
 
-const UpdateProfile = () => {
-  return (
-      <StyledForm>
-          <StyledCard>
-              Edit Profile
-        </StyledCard>
-    </StyledForm>
-  );
+const UpdateProfile = ({
+    setUpdatedUser,
+    updatedUser,
+    setUser,
+    id,
+    isLoading,
+    error,
+    sessionToken
+}) => {
+    
+    const history = useHistory();
+
+    if (isLoading) {
+        return <p>Loading...</p>;
+      }
+    
+      if (error) {
+        return <p>An error has occurred...</p>;
+      }
+  
+    const handleChange = event => {
+        const { name, value } = event.target;
+        setUpdatedUser({ ...updatedUser, [name]: value });
+    }
+    
+    const submitForm = async () => {
+        try {
+          await updateUserById(id, updatedUser, sessionToken.token);
+          setUser(updatedUser);
+          history.push(`/users/${id}`);
+        } catch (error) {
+          console.log(error);
+        }
+    };
+
+    return (
+        <StyledForm>
```

Comment:
I think you need to fix your prettier settings, the indentation and formatting is quite inconsistent across different files